.circleci/config.yml

```diff
@@ -94,6 +94,14 @@ workflows:
               only: /v[0-9]+.[0-9]+.[0-9]+(-.+)?/
             branches:
               ignore: /.*/
+      - check-bundle-size:
+          requires:
+            - build
+            - deploy-release
```

Comment:
>A list of jobs that must succeed for the job to start. Note: When jobs in the current workflow that are listed as dependencies are not executed (due to a filter function for example), their requirement as a dependency for other jobs will be ignored by the requires option.

Could you try to trigger a build that has both dependencies met to confirm this works as expected?

In order to test that, I would change the filter on line 94 to pass always, and replace every upload in the deploy-release by a dummy 'echo' instead.

```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index 3f52fe04e..27bf9e02d 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -75,7 +75,7 @@ workflows:
             - test-expressions
           filters:
             tags:
-              only: /v[0-9]+.[0-9]+.[0-9]+(-.+)?/
+              only: /.*/
             branches:
               only:
                 - main
@@ -287,7 +287,7 @@ jobs:
           name: Deploy release
           command: |
             function upload {
-                aws s3 cp --acl public-read --content-type $2 dist/$1 s3://mapbox-gl-js/$CIRCLE_TAG/$1
+                echo aws s3 cp --acl public-read --content-type $2 dist/$1 s3://mapbox-gl-js/$CIRCLE_TAG/$1
             }
 
             upload mapbox-gl.js     application/javascript

```

I'm mainly curious about seeing a build actually running 'Check build file for correct SDK version'.

Reply:
We can't test this fully on CI because the `$CIRCLE_TAG` environment variable is only set when CI is running against a tag, so the test would always fail on a branch. But testing the script locally confirms that it works as expected.

<img width="488" alt="Screen Shot 2021-01-20 at 2 34 58 PM" src="https://user-images.githubusercontent.com/4523080/105249411-ebf01300-5b2c-11eb-9c34-51ddd95ff186.png">